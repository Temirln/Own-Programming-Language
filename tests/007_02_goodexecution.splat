program
    myFunction(param1: Integer,
               param2: Boolean) : void is
    begin
        if param2 then
            print "param2 is true";
        else
            print "param2 is false";
        end if;
		return;
        print "myFunction() called";
    end;
    myInteger: Integer;
    result: Integer;
    myBoolean: Boolean;
begin
    myBoolean := true;
    print "program started";
    print_line;
    myFunction(myInteger, myBoolean);
end;