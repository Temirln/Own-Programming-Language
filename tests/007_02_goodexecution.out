program started
param2 is true